validator/sawtooth_validator/exceptions.py

```diff
@@ -72,3 +72,7 @@ class PossibleForkDetectedError(Exception):
     through the block store.
     """
     pass
+
+
+class ProcessorQueuesFullError(Exception):
```

Comment:
Maybe rename this `NoProcessorVacancyError` to keep with the language used in the code.